src/components/console-overrides.scss

```diff
@@ -0,0 +1,11 @@
+.pf-c-table.hosts-table {
```

Comment:
To avoid applying the styles to `hosts-table` class, we could update `HostsTable` component in ui-lib to take a custom className as a prop. Then we could use custom classname to define overrides on if needed.

Reply:
that would be my preference

Reply:
Done